src/components/dropdown/Dropdown.tsx

```diff
@@ -99,11 +99,11 @@ export class Dropdown<T = LabelValue> extends React.Component<
   };
 
   handleIsOptionDisabled = this.props.isOptionDisabled
-    ? (option: any) => this.props.isOptionDisabled(option.data)
+    ? (option: T) => this.props.isOptionDisabled(option)
```

Comment:
If I am not mistaken, we still need to pass option.data to `this.props.isOptionDisabled` (?)

Reply:
Previously, we assumed that there was a 'data' object in the option, but it was not always the case (If for example, the user instantiate a dropdown with Dropdown<HisType>)
Now, we send the full option and not just a subset of the option. This was used in the TimePicker, I updated the code. And I'm going to list this in the Breaking changes of the v3

Reply:
Ok we are not only replacing 'any' but also changing the type expected by the prop. This will directly impact NEXUS (and other consumers)